continuous_integration/environment-3.10.yaml

```
@@ -36,10 +37,14 @@ dependencies:
   - sortedcollections
   - tblib
   - toolz
-  - tornado=6
+  - tornado
   - zict  # overridden by git tip below
   - zstandard
   - pip:
       - git+https://github.com/dask/dask
+      - git+https://github.com/dask/s3fs
       - git+https://github.com/dask/zict
+      - git+https://github.com/intake/filesystem_spec
```

Minor tweak: `filesystem_spec` moved from the `intake` to `fsspec` GitHub org recently 

```suggestion
      - git+https://github.com/fsspec/filesystem_spec
```